mygfa/mygfa/mygfa.py

```diff
@@ -43,31 +43,62 @@ def __str__(self) -> str:
         return "\t".join([self.name, str(self.low), str(self.high), self.new])
 
 
+@dataclass
+class Strand:
+    """A strand is a string that contains only A, T, G, C, or N."""
+
+    strand: str
```

Comment:
One other option—and I'm not 100% sure this is a good idea—is to make `Strand` a subclass of `str` rather than a dataclass with one field. That would avoid the need to access `self.strand` in a few places; you could just use `self`. And we wouldn't need `__str__` or `__len__`; those would already be implemented for us.

Reply:
Oh I like it a lot! Done ~~